src/pdb.rs

```diff
@@ -97,6 +96,22 @@ impl<'s, S: Source<'s> + 's> PDB<'s, S> {
         TypeInformation::parse(stream)
     }
 
+    /// Retrieve the `TypeInformation` for this PDB.
```

Comment:
`id_information()` and `type_information()` shouldn't both have the same doc comment, despite their implementation similarities.

Reply:
Done.